FreeRTOS/FreeRTOS-Plus-TCP
# IAR warning fixes

Description
-----------
This PR fixes the warnings about the usage of forward declarations in the IAR compiler by supressing the warning.

Test Steps
-----------
Run demo project built on IAR

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
